cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -280,16 +281,6 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
         }
 
         val counts = evaluatorRun.violations.groupingBy { it.severity }.eachCount()
-
-        val errorCount = counts[Severity.ERROR] ?: 0
-        val warningCount = counts[Severity.WARNING] ?: 0
-        val hintCount = counts[Severity.HINT] ?: 0
-
-        if (errorCount > 0 || warningCount > 0) {
-            println("Found $errorCount errors, $warningCount warnings, $hintCount hints.")
-            throw ProgramResult(2)
-        }
-
-        println("Found $hintCount hints only.")
+        if (printSeverityStats(counts, Severity.HINT)) throw ProgramResult(2)
```

Comment:
This reads like `ProgramResult(2)` is thrown if the stats could be printed, I'd prefer to find a better name for the function but don't have a good suggestion. Another option would be to split `printSeverityStats` and the `hasSevereIssues` check into separate functions.

Reply:
Or how about simply moving the `throw ProgramResult(2)` into the function (as that's what all callers currently do) and not return anything?

Reply:
I did that now in the new iteration.

Reply:
I still find it very unexpected that a function with the name "print..." has such a side effect, but it's merged now, so be it.

Reply:
I'm more than happy to make a follow-up PR to rename, it's not carved in stone. But the alignment of the exit code was more important to me right now. Do you have a proposal for a name by now?